easybuild/easyconfigs/j/Java/Java-1.8.0_72.eb

```diff
@@ -0,0 +1,41 @@
+# Built with EasyBuild version 2.5.0 on 2016-01-21_18-39-33
+name = 'Java'
+version = "1.8.0_72"
+
+homepage = 'http://java.com/'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download the tar.gz directly from http://www.oracle.com/technetwork/java/javase/downloads/index.html
+(vp, vs) = version.split('_')
+altver = '%su%s' % (vp.split('.')[1], vs)
+sources = ['jdk-%s-linux-x64.tar.gz' % altver]
+
+moduleclass = 'lang'
+
+# Build statistics
+buildstats = [{
```

Comment:
Will do - directly copied from Simon's easyconfig.

I handled it separately, as you saw.